recipes-support/snapd/snapd-go.inc

```diff
@@ -0,0 +1,72 @@
+inherit go-mod
+
```

Comment:
Do we need to put `do_compile[network] = "1"` anywhere?

Reply:
no, the recipe uses snapd with vendored dependencies, so everything is included already in the tarball